cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsRdsDbParameterGroupViewTest.java

```diff
@@ -131,4 +133,34 @@ void getDBParameterGroupFamilyTestWhenPgSqlAndValidVersion(String testCaseName,
         assertThat(underTest.getDBParameterGroupFamily()).isEqualTo(familyExpected);
     }
 
+    @Test
+    void templateAndDefaultPostgresVersionIsMatching() throws IOException {
+        assertThat(AwsRdsDbParameterGroupView.DEFAULT_FAMILY_VERSION).isEqualTo(getDefaultEngineVersionFromTemplate());
+    }
+
+    private String getDefaultEngineVersionFromTemplate() throws IOException {
+        String template = getTemplate();
+        Pattern defaultEngineVersion = Pattern.compile(ENGINE_VERSION_PARAMETER_REGEX);
```

Comment:
Erm, I meant making the whole `Pattern` instance a constant. 😛

Reply:
Oh, ok. got it